.ci/travis/travis.sh

```diff
@@ -461,6 +461,31 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-error-test-josm)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  mkdir -p .ci-temp
+  cd .ci-temp
+  ANT_VERSION=1.10.7
+  wget http://www-us.apache.org/dist//ant/binaries/apache-ant-${ANT_VERSION}-bin.zip
+  unzip -q apache-ant-${ANT_VERSION}-bin.zip
```

Comment:
please add `-o` to avoid:
```
2020-05-09 17:20:06 (1.61 MB/s) - ‘apache-ant-1.10.7-bin.zip.1’ saved [10186279/10186279]

replace apache-ant-1.10.7/bin/ant? [y]es, [n]o, [A]ll, [N]one, [r]ename: y
replace apache-ant-1.10.7/bin/antRun? [y]es, [n]o, [A]ll, [N]one, [r]ename: y
replace apache-ant-1.10.7/bin/antRun.pl? [y]es, [n]o, [A]ll, [N]one, [r]ename: y
replace apache-ant-1.10.7/bin/complete-ant-cmd.pl? [y]es, [n]o, [A]ll, [N]one, [r]ename: A
Apache Ant(TM) version 1.10.7 compiled on September 1 2019
....
```

Reply:
I still do not see '-o' option for unzip.
Please explain.

Reply:
Please do not resolve review items, I will resolve them as ack that fix is properly done. Please just reply "done" in each review item.